app/services/ocean_service.rb

```diff
@@ -0,0 +1,66 @@
+class OceanService
+  ITERATIONS = 8
+  SEED_POLYGON_OPTIONS = { sides: 5, center: [500, 500], radius: 100 }
+
+  def call
+    inline = create_polygon
+    outline = Geometry::Polygon.new(*inline.vertices)
+    {
+      islands:  [
+        {
```

Comment:
extract a method that returns only one island hash, so it is easier lateron to do multiple island later.